standalone/runtime/Cargo.toml

```diff
@@ -14,155 +14,169 @@ targets = ["x86_64-unknown-linux-gnu"]
 substrate-wasm-builder = "4.0.0"
 
 [dependencies]
-codec = { package = "parity-scale-codec", version = "2.3.0", default-features = false, features = ["derive", "max-encoded-len"] }
-hex-literal = { version = '0.3.1', optional = true }
-log = { version = "0.4.14", default-features = false }
-scale-info = { version = "1.0.0", default-features = false, features = ["derive"] }
-serde = { version = "1.0.101", optional = true, features = ["derive"] }
-static_assertions = "1.1.0"
+codec = {package = "parity-scale-codec", version = "2.3.0", default-features = false, features = ["derive", "max-encoded-len"]}
+# This getrandom package was having issues been compiled to wasm, that's why this override is here
+getrandom = {version = "0.2", features = ["js"]}
```

Comment:
This should never be in a runtime, `getrandom` doesn't compile to WASM that Substrate does afaik.

Reply:
The error you get often from the compiler is very misleading. When you do get WASM compilation errors it means that some other dependency is pulling in `std` features. The solution is not to add this from my experience.